nixos/modules/services/matrix/mautrix-telegram.nix

```diff
@@ -25,6 +25,7 @@ in {
             hostname = "0.0.0.0";
             port = 8080;
             address = "http://localhost:${toString port}";
+            software = "standard";
```

Comment:
This new key should be under `homeserver` instead of `appservice`.
(See mautrix_telegram/example-config.yaml).

Reply:
Sorry, didn't realize -- fixed.